src/ui/zcl_abapgit_gui_page_commit.clas.abap

```diff
@@ -392,6 +392,7 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_COMMIT IMPLEMENTATION.
 
     CREATE OBJECT ro_html.
 
+    ro_html->zif_abapgit_html~set_title( cl_abap_typedescr=>describe_by_object_ref( me )->get_relative_name( ) ).
```

Comment:
does this work in the compiled version?

Reply:
Yes

![image](https://user-images.githubusercontent.com/15635498/82748423-61db8900-9daa-11ea-8d8a-907f08d6c45e.png)

P.S. found one forgotten place, will add a small commit in a moment

Reply:
cool

Reply:
ok, done. Forgot to add title in hotkeys scripts because it was not unified in the first place